WordPress/Classes/Models/Blog.m

```diff
@@ -235,25 +238,55 @@ - (NSArray *)sortedCategories
     return [[self.categories allObjects] sortedArrayUsingDescriptors:sortDescriptors];
 }
 
+- (NSArray *)sortedPostFormats
+{
+    if ([self.postFormats count] == 0) {
+        return @[];
+    }
+    NSMutableArray *sortedFormats = [NSMutableArray arrayWithCapacity:[self.postFormats count]];
+ 
+    if (self.postFormats[@"standard"]) {
+        [sortedFormats addObject:@"standard"];
+    }
+    [self.postFormats enumerateKeysAndObjectsUsingBlock:^(id key, id obj, BOOL *stop) {
+        if (![key isEqual:@"standard"]) {
+            [sortedFormats addObject:key];
+        }
+    }];
+
+    return [NSArray arrayWithArray:sortedFormats];
+}
+
 - (NSArray *)sortedPostFormatNames
 {
     NSMutableArray *sortedNames = [NSMutableArray arrayWithCapacity:[self.postFormats count]];
 
-    if ([self.postFormats count] != 0) {
-        id standardPostFormat = [self.postFormats objectForKey:@"standard"];
-        if (standardPostFormat) {
-            [sortedNames addObject:standardPostFormat];
-        }
-        [self.postFormats enumerateKeysAndObjectsUsingBlock:^(id key, id obj, BOOL *stop) {
-            if (![key isEqual:@"standard"]) {
-                [sortedNames addObject:obj];
-            }
-        }];
+    for (NSString *key in self.sortedPostFormats) {
+        [sortedNames addObject:self.postFormats[key]];
     }
 
     return [NSArray arrayWithArray:sortedNames];
 }
 
+- (NSString *)defaultPostFormatText
+{
+    return [self postFormatTextForCode:self.defaultPostFormat];
+}
+
+- (NSString *)postFormatTextForCode:(NSString *)postFormatCode
+{
+    NSDictionary *allFormats = self.postFormats;
+    NSString *formatText = postFormatCode;
+    if (postFormatCode && allFormats[postFormatCode]) {
+        formatText = allFormats[postFormatCode];
```

Comment:
"Code" was a bit confusing to me at first, I think the more _WordPress_ term for this is "slug" (e.g. https://codex.wordpress.org/Function_Reference/get_post_format_string)